wgengine/netstack/netstack.go

```diff
@@ -1295,7 +1326,7 @@ func (ns *Impl) ExpVar() expvar.Var {
 
 	// Global metrics
 	stats := ns.ipstack.Stats()
-	m.Set("gauge_dropped_packets", expvar.Func(func() any {
+	m.Set("counter_dropped_packets", expvar.Func(func() any {
```

Comment:
So, silly question but... I used `gauge_` because these values aren't reset on read (i.e. if you query `counter_dropped_packets` once and get 10, the next time you query it you'll get >=10). How does that interact with our metrics scraping?

Reply:
Ah, great, and thanks for the link; appreciate the cleanup 🙏